src/Controller/SnapshotAdminController.php

```diff
@@ -35,51 +30,10 @@ final class SnapshotAdminController extends CRUDController
     public static function getSubscribedServices(): array
     {
         return [
-            'sonata.page.manager.page' => PageManagerInterface::class,
             'sonata.page.manager.snapshot' => SnapshotManagerInterface::class,
-            'sonata.page.service.create_snapshot' => CreateSnapshotService::class,
         ] + parent::getSubscribedServices();
     }
 
-    public function createAction(Request $request): Response
```

Comment:
this create was trying to replace the original, which causes issues like this one: https://github.com/sonata-project/SonataPageBundle/issues/1453

cc/ @eerison 

Also this create for some reason use a custom form type, instead of relying on the admin. IMHO this is a bad idea, and that's why I tried to remove it. So we can fix both issues, because this PR also solves the other issue: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1210757295